src/main/java/domain/Cards.java

```diff
@@ -0,0 +1,53 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cards {
+
+	private static final int BLACK_JACK_SCORE = 21;
+	private static final int ACE_OFFSET = 10;
+
+	private final List<Card> cards;
+
+	public Cards() {
+		this.cards = new ArrayList<>();
+	}
```

Comment:
```suggestion
	public Cards(List<Card> cards) {
		this.cards = cards;
	}
```
이렇게 쓰는게 더 다양한 방법으로 쓸 수 있지 않을까요?